### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by scseeman**

### CODE REVIEW

File: aisynphys/ui/notebook.py [hunk 1]
```diff
@@ -251,40 +257,36 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
...
-        'stp_induction_50hz':                 ('← Facilitating  Depressing →',      '',    1,    1,     [db.Dynamics.stp_induction_50hz],                  'bwr',         False,  (-0.5, 0.5),   "%0.2f"),
-        'stp_recovery_250ms':                 ('← Over-recovered  Not recovered →', '',    1,    1,     [db.Dynamics.stp_recovery_250ms],                  'bwr',         False,  (-0.2, 0.2),   "%0.2f"),
-        'stp_recovery_single_250ms':          ('← Over-recovered  Not recovered →', '',    1,    1,     [db.Dynamics.stp_recovery_single_250ms],           'bwr',         False,  (-0.2, 0.2),   "%0.2f"),
-        'pulse_amp_first_50hz':               ('1st PSP Amplitude @ 50Hz',          '',    1e3,  1,     [db.Dynamics.pulse_amp_first_50hz],                'bwr',         False,  (-1.5, 1.5),   "%0.2f\nmV"),
-        'pulse_amp_stp_initial_50hz':         ('2nd PSP Amplitude @ 50Hz',          '',    1e3,  1,     [db.Dynamics.pulse_amp_stp_initial_50hz],          'bwr',         False,  (-1.5, 1.5),   "%0.2f\nmV"),
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:270]: Looks like some things were lost in this merge.

File: aisynphys/ui/notebook.py [hunk 2]
```diff
@@ -251,40 +257,36 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
...
-        'coupling_coeff_pulse':               ('Coupling Coefficient',              '',    1,    1,     [db.GapJunction.coupling_coeff_pulse],             'viridis',     False,  (0, 1),        "%0.2f"),
-        'variability_resting_state':          ('log(Resting state variance)',       '',    1,    1,     [db.Dynamics.variability_resting_state],           'viridis',     False,  (-1, 1),       "%0.2f"),
-        'variability_second_pulse_50hz':      ('log(second pulse variance)',         '',    1,    1,    [db.Dynamics.variability_second_pulse_50hz],       'viridis',     False,  (-1, 1),       "%0.2f"),
-        'variability_stp_induced_state_50hz': ('log(STP induced variance)',         '',    1,    1,     [db.Dynamics.variability_stp_induced_state_50hz],  'viridis',     False,  (-1, 1),       "%0.2f"),
+        #                                     name                                  unit   scale alpha  db columns                                        map fn       colormap       log     clim           text format
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:260]: no need to re-add the map-fn column; that has been removed.

File: aisynphys/ui/notebook.py (hunk 1 above)
Comment (scseeman) [reply aisynphys/ui/notebook.py:270]: do you remember which ones?

Comment (scseeman) [reply aisynphys/ui/notebook.py:270]: I just copied back the one from master, my only change was the map_fn, and name change to variance

**Pull request merged**